snapshots/ihme_gbd/2024-06-10/cause_hierarchy.csv.dvc

```diff
@@ -0,0 +1,31 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+# Download the file from: https://ghdx.healthdata.org/record/global-burden-disease-study-2021-gbd-2021-cause-rei-and-location-hierarchies
+# Select - Cause Hierarchy [XLSX]
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Global Burden of Disease - Cause Hierarchy
+    description: |-
+      The hierarchy of diseases and injuries included in the Global Burden of Disease Study.
+    date_published: "2024-05-17"
+    title_snapshot: Global Burden of Disease Study - Cause Hierarchy
```

Comment:
just double checking, since `title` and `title_snapshot` are almost equal. This is expected?

This is personal: I tend to like reserving `-` for `title_snapshot`, so I'd go with "Global Burden of Disease, Cause of Hierarchy" as a `title`. But, again, this is personal.